src/kirby/components/button/button.component.scss

```diff
@@ -13,18 +13,25 @@ button {
   background-color: var(--primary);
   border-color: var(--primary);
   border-radius: var(--kirby-button-border-radius);
-  border: 1px solid;
+  border-width: 1px;
+  border-style: solid;
   color: var(--on-primary);
```

Comment:
Maybe something more specific than just `--on-primary`, to avoid clashing with other components/frameworks?
+ Add suffix (`-color`, `-background-color`, `-border-color`, `border-radius` etc)
E.g. `--kirby-on-primary-color`